content/collections.md

```diff
@@ -0,0 +1,339 @@
+
+    return super(list, callback);
+  }
+}
+
+Lists = new ListsCollection('Lists');
+```
+
+### Writing "hooks"
+
+The technique above can also be used to provide a location to "hook" extra functionality into the collection. For instance, when removing a list, we *always* want to remove all of its todos at the same time.
+
+To do so in an easy to understand way, we can again subclass, overriding the `remove()` method:
+
+```js
+class ListsCollection extends Mongo.Collection {
+  ...
+  remove(selector, callback) {
+    Package.todos.Todos.remove({listId: selector});
+    return super(selector, callback);
+  }
+}
+```
+
+This technique has a couple of downsides:
+
+  1. Mutators can get very long when you want to hook in multiple times
+  2. Sometimes a single piece of functionality can be spread over multiple mutators
+  3. It can be a challenge to write a hook in a completely general way (that covers every possible selector and modifier), and it may not be necessary for your application (because perhaps you only ever call that mutator in one way).
+
+A way to deal with points 1. and 2. is to separate out the set of hooks into their own module, and simply use the mutator as a point to call out to that module in a sensible way. We can see in the "Forms and Methods" chapter an example of how we do that in the list and todo denormalizers mentioned above.
+
+Point 3. can usually be resolved by placing the hook in the *Method* that calls the mutator, rather than the hook itself. Although this is an imperfect compromise (as we need to be careful if we ever add another Method that calls that mutator in the future), it is better than writing a bunch of code that is never actually called (which is guaranteed to not work!), or giving the impression that your hook is more general that it actually is.
+
+## Changing your schema: using data migrations
+
+As we discussed above, trying to predict all future requirements of your data schema ahead of time is of course impossible. So inevitably as a project matures, there'll come a time when you need to change the schema of its data. To do so safely, you need to be careful about how you make the changes.
+
+### Writing migrations
+
+A useful package for writing migrations is the [`percolate:migrations`](https://atmospherejs.com/percolate/migrations) package. Suppose, as an example, that we wanted to add a `list.todoCount` field, and ensure that it was set for all existing lists. Then we might write:
+
+```js
+Migrations.add({
+  version: 1,
+  up() => {
+    Lists.find({todoCount: {$exists: false}}).forEach(list => {
+      const todoCount = Todos.find({listId: list._id})).count();
+      Lists.update(list._id, {$set: {todoCount}});
+    });
+  },
+  down() {
+    Lists.update({}, {$unset: {todoCount: true}});
+  }
+});
+```
+
+This migration, which is sequenced to be the first migration to run over the database, will, when called, bring each list up to date with the current todo count.
+
+### Running migrations
```

Comment:
I added something about development migrations.

Re: docs -- the original mention of the package is a link. Do you think I should call it out more specifically?